src/api/hocrrenderer.cpp

```diff
@@ -500,28 +500,30 @@ TessHOcrRenderer::TessHOcrRenderer(const char *outputbase, bool font_info)
 }
 
 bool TessHOcrRenderer::BeginDocumentHandler() {
+  SetContentType("application/xhtml+xml");
+  return true;
```

Comment:
I don't understand this code. Where is `SetContentType` defined? And why do you add a `return` statement which makes the following code useless?